src/AzureExtension/DevBox/Helpers/WaitingForUserAdaptiveCardSession.cs

```diff
@@ -21,6 +22,14 @@ public class WaitingForUserAdaptiveCardSession : IExtensionAdaptiveCardSession2,
 
     private ManualResetEvent _launchEvent;
 
+    private const string _waitingForUserSessionState = "WaitingForUserSession";

```

Comment:
Coding guidelines: Normally constant names don't start with "_": "[Use PascalCase for constant names, both fields and local constants.](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#:~:text=Use%20PascalCase%20for%20constant%20names%2C%20both%20fields%20and%20local%20constants.)"